### DISCUSSION TITLE
Dart/Flutter + Supabase Support

### DISCUSSION THREAD

**Discussion by jtkeyva** [electric-sql/electric#555] [Feature requests]
Hello,

This could be the missing link to an amazing setup. I see the Dart/Flutter package https://pub.dev/packages/electricsql is a WIP.

I also saw in Discord someone saying it will be a couple weeks to find an alternative to needing Superuser access and that Supabase integration was doable in theory.

Might want to poke around here for ideas/solutions:
https://docs.powersync.co/integration-guides/supabase-+-powersync

Do you see this in the near future?

Thanks

**Comment by thruflo** [comment]
Hey, yes. As per discord thread, we will unblock this level of Supabase integration shortly. It’s just a temporary permission level / packaging issue.

**Comment by jtkeyva** [comment]
Wow that's great to hear! Can't wait to try it out

**Comment by balegas** [comment]
Turned this issue into a feature request discussion

**Comment by jtkeyva** [comment]
Hello @balegas is there any movement on this?

**Comment by balegas** [comment]
Hi, we still need some work before adding supabase support.

**Reply by balegas** [reply]
We shipped the main blocker on permissions, the proxy, but we found a few more issues with permission granting that is holding off working with some hosted platforms like Supabase. We’re looking more systematically to be deployable acrosss hosted platforms, but for the next release this year we don’t expect to fully unlock Supabase support.

**Reply by jtkeyva** [reply]
@dshukertjr this is a very special opportunity for supabase to stay ahead of the curve. offline is the new normal and this highly requested feature for supabase from what i've seen.

any help greatly appreciated

**Comment by kaikun213** [comment]
Thanks a lot for adding the integration! (https://youtube.com/watch?v=WhRBvJ4cUWk)
QQ Regarding deployment: How would you deploy the sync-service with supabase for optimal performance? (ie close to the DB/users) The guide currently only points to local deployment

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
